public/partials/commodity_edit.html

```diff
@@ -0,0 +1,57 @@
+<div ng-controller="NewCommodityCtrl as ctrl">
+    <md-content>
+
+        <md-input-container flex>
+            <label>Crop Name</label><input disabled="disabled" min="0" ng-required="true" ng-model="commodity.cr_crop.cr_crop_name" type="text"/>
```

Comment:
@nick22891 this will not work.  Check the implementation of `commodity_new.html`.